frontend/views/components/ListItem.vue

```diff
@@ -76,31 +67,39 @@ export default {
 .c-item-link {
   display: flex;
   align-items: center;
-  padding: 0 $spacer;
-  height: $spacer-lg;
+  padding: 0 1.25rem;
```

Comment:
shouldn't it better to use `$spacer*1.25` so we avoid raw values?

Reply:
Please no

Reply:
ok.

Reply:
I'm still wondering why no, REALLY. $spacer is 1rem, so this a relative size to it (*1.25)... it's the same as not using $spacer and replace it by 1rem.

Reply:
If you want..